download/download.go

```diff
@@ -48,7 +49,8 @@ func getURLs(client *http.Client, src string) ([]string, error) {
 			return
 		}
 		if strings.HasSuffix(h, ".zip") {
-			urls = append(urls, h)
+			r := regexp.MustCompile(`(http)(s?)(//)`)
+			urls = append(urls, r.ReplaceAllString("http// https//", `$1$2$3`))
```

Comment:
Não rodei o código, mas tenho duas dúvidas aqui:

1. Não seria `h` ao invés de `"http// https//"`?
2. Não teria que incluir o `:` faltante, algo como `"$1$2:$3` ao invés de `"$1$2$3"`?

```suggestion
			urls = append(urls, r.ReplaceAllString(h, `$1$2:$3`))
```

Reply:
com o ponto temos esse erro

```
Error: Error sending a HTTP HEAD request to http://:200.152.38.155/CNPJ/K3241.K03200Y0.D11113.SOCIOCSV.zip: parse "http://:200.152.38.155/CNPJ/K3241.K03200Y0.D11113.SOCIOCSV.zip": invalid port ":200.152.38.155" after host
```